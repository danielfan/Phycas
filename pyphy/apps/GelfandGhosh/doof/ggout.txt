Simulation conditions:
  no. sites   : 2000
  model tree  : (1:0.2,2:0.02,(3:0.2,4:0.02):0.02)
  subst. model: HKY85+G
  kappa       : 4.0
  shape       : 0.2
  piA         : 0.2
  piC         : 0.3
  piG         : 0.3
  piT         : 0.2


Gelfand-Ghosh calculation for HKY analysis:
       k           Pm           Gm           Dm
     0.5    217.10875     60.46673    277.57549
     1.0    217.10875     95.91029    313.01905
     2.0    217.10875    139.75354    356.86230

Gelfand-Ghosh calculation for HKY+G analysis:
       k           Pm           Gm           Dm
     0.5    216.36597     60.36145    276.72742
     1.0    216.36597     95.83072    312.19670
     2.0    216.36597    139.75122    356.11719

Gelfand-Ghosh calculation for HKY+FLEX analysis:
       k           Pm           Gm           Dm
     0.5    215.66263     60.79076    276.45339
     1.0    215.66263     96.38228    312.04491
     2.0    215.66263    140.38683    356.04945

Penalty rankings (best listed first):
     HKYFLEX    215.66263
        HKYg    216.36597
         HKY    217.10875

For k = 0.5:

  Goodness-of-fit rankings (best listed first):
        HKYg     60.36145
         HKY     60.46673
     HKYFLEX     60.79076

  Overall rankings (best listed first):
     HKYFLEX    276.45339
        HKYg    276.72742
         HKY    277.57549

For k = 1.0:

  Goodness-of-fit rankings (best listed first):
        HKYg     95.83072
         HKY     95.91029
     HKYFLEX     96.38228

  Overall rankings (best listed first):
     HKYFLEX    312.04491
        HKYg    312.19670
         HKY    313.01905

For k = 2.0:

  Goodness-of-fit rankings (best listed first):
        HKYg    139.75122
         HKY    139.75354
     HKYFLEX    140.38683

  Overall rankings (best listed first):
     HKYFLEX    356.04945
        HKYg    356.11719
         HKY    356.86230
