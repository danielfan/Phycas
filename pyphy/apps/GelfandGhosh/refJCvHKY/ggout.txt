Simulation conditions:
  no. sites   : 1000
  model tree  : (1:0.1,2:0.1,(3:0.1,4:0.1):0.1)
  subst. model: JC69

Gelfand-Ghosh calculation for JC analysis:
       k           Pm           Gm           Dm
     0.5    207.88300     44.80524    275.09085
     1.0    207.88300     52.39224    312.66748
     2.0    207.88300     49.97153    357.79758

Gelfand-Ghosh calculation for HKY analysis:
       k           Pm           Gm           Dm
     0.5    209.62688     43.91852    275.50465
     1.0    209.62688     51.22480    312.07648
     2.0    209.62688     48.72790    355.81057

Gelfand-Ghosh calculation for HKY+G analysis:
       k           Pm           Gm           Dm
     0.5    226.64799     46.20326    295.95288
     1.0    226.64799     54.76347    336.17493
     2.0    226.64799     52.98799    385.61195

Gelfand-Ghosh calculation for HKY+G+I analysis:
       k           Pm           Gm           Dm
     0.5    218.52038     44.01363    284.54083
     1.0    218.52038     51.94057    322.40152
     2.0    218.52038     49.97423    368.44306

Gelfand-Ghosh calculation for GTR analysis:
       k           Pm           Gm           Dm
     0.5    219.59310     42.89726    283.93899
     1.0    219.59310     50.22646    320.04603
     2.0    219.59310     47.94079    363.41547

Penalty rankings (best listed first):
          JC    207.88300
         HKY    209.62688
       HKYgi    218.52038
         GTR    219.59310
        HKYg    226.64799

For k = 0.5:

  Goodness-of-fit rankings (best listed first):
         GTR     42.89726
         HKY     43.91852
       HKYgi     44.01363
          JC     44.80524
        HKYg     46.20326

  Overall rankings (best listed first):
          JC    275.09085
         HKY    275.50465
         GTR    283.93899
       HKYgi    284.54083
        HKYg    295.95288

For k = 1.0:

  Goodness-of-fit rankings (best listed first):
         GTR     50.22646
         HKY     51.22480
       HKYgi     51.94057
          JC     52.39224
        HKYg     54.76347

  Overall rankings (best listed first):
         HKY    312.07648
          JC    312.66748
         GTR    320.04603
       HKYgi    322.40152
        HKYg    336.17493

For k = 2.0:

  Goodness-of-fit rankings (best listed first):
         GTR     47.94079
         HKY     48.72790
          JC     49.97153
       HKYgi     49.97423
        HKYg     52.98799

  Overall rankings (best listed first):
         HKY    355.81057
          JC    357.79758
         GTR    363.41547
       HKYgi    368.44306
        HKYg    385.61195
