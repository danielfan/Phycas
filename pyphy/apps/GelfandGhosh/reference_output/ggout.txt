Simulation conditions:
  no. sites   : 1000
  model tree  : (1:0.5,2:0.01,(3:0.5,4:0.01):0.01)
  subst. model: HKY85+G
  kappa       : 4.0
  shape       : 0.2
  piA         : 0.2
  piC         : 0.3
  piG         : 0.3
  piT         : 0.2


Gelfand-Ghosh calculation for HKY analysis:
       k           Pm           Gm           Dm
     0.5    139.87433     78.91531    218.78964
     1.0    139.87433    119.11952    258.99385
     2.0    139.87433    164.59918    304.47351

Gelfand-Ghosh calculation for HKY+G analysis:
       k           Pm           Gm           Dm
     0.5    162.30560     43.78876    206.09436
     1.0    162.30560     68.23733    230.54293
     2.0    162.30560     97.65548    259.96107

Gelfand-Ghosh calculation for HKY+FLEX analysis:
       k           Pm           Gm           Dm
     0.5    162.97068     42.37606    205.34673
     1.0    162.97068     65.89991    228.87059
     2.0    162.97068     94.10504    257.07571

Penalty rankings (best listed first):
         HKY    139.87433
        HKYg    162.30560
     HKYFLEX    162.97068

For k = 0.5:

  Goodness-of-fit rankings (best listed first):
     HKYFLEX     42.37606
        HKYg     43.78876
         HKY     78.91531

  Overall rankings (best listed first):
     HKYFLEX    205.34673
        HKYg    206.09436
         HKY    218.78964

For k = 1.0:

  Goodness-of-fit rankings (best listed first):
     HKYFLEX     65.89991
        HKYg     68.23733
         HKY    119.11952

  Overall rankings (best listed first):
     HKYFLEX    228.87059
        HKYg    230.54293
         HKY    258.99385

For k = 2.0:

  Goodness-of-fit rankings (best listed first):
     HKYFLEX     94.10504
        HKYg     97.65548
         HKY    164.59918

  Overall rankings (best listed first):
     HKYFLEX    257.07571
        HKYg    259.96107
         HKY    304.47351
